[FEATURE] Add UserSettings JavaScript Module (#1052)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/Hooks/Events/Setup/AddJavaScriptModulesEvent.rst

.. include:: ../../../../Includes.txt

.. _AddJavaScriptModulesEvent:

=========================
AddJavaScriptModulesEvent
=========================

> JavaScript events in custom User Settings Configuration options shall not be placed as inline JavaScript anymore, but utilize a dedicated JavaScript module to handle custom events.
> 

Reviewer: ```suggestion
JavaScript events in custom User Settings Configuration options should no longer be placed as inline JavaScript. Instead, use a dedicated JavaScript module to handle custom events.

```
"Shall" can be ambiguous. I guess at the intended meaning.
